src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -8,26 +8,28 @@
 import mate.academy.hibernate.relations.service.CountryService;
 import mate.academy.hibernate.relations.service.MovieService;
 import mate.academy.hibernate.relations.util.HibernateUtil;
+import mate.academy.hibernate.relations.util.Injector;
 import org.hibernate.SessionFactory;
 
 public class Main {
     public static void main(String[] args) {
-        // use this session factory when you will initialize service instances
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
+        Injector injector = new Injector(sessionFactory);
```

Comment:
Ensure that the `Injector` class is correctly implemented to provide the necessary services. The constructor and methods should be properly defined to initialize and return the service instances as expected.